src/components/state-history-chart-line.js

```diff
@@ -181,23 +181,65 @@ class StateHistoryChartLine extends LocalizeMixin(PolymerElement) {
               state.attributes.target_temp_low
         );
 
-        addColumn(name + " current temperature", true);
+        addColumn(
+          [
+            name,
+            this.hass
+              .localize("ui.card.climate.current_temperature")
+              .toLowerCase(),
+          ].join(" "),
+          true
+        );
         if (hasHeat) {
-          addColumn(name + " heating", true, true);
+          addColumn(
+            [name, this.hass.localize("ui.card.climate.heating")].join(" "),
```

Comment:
```suggestion
            `${name} ${this.hass.localize("ui.card.climate.heating")}`,
```

Reply:
Yeah you are completely right, that simplifies it immensely 👍

Reply:
> And we should pass name as a variable to the translation string.

@bramkragten so both or just template literals?

Reply:
Can't do both ;-) just add it as a variable, that's nicer.

Reply:
Yeah I agree ;-) sure thing.